config/chainbridge-runtime-config.cere.stage.js

```diff
@@ -28,8 +28,8 @@ window.__RUNTIME_CONFIG__ = {
         tokens: [
           {
             address: "0xd111d479e23A8342A81ad595Ea1CAF229B3528c3",
-            name: cereTokenName,
-            symbol: cereTokenSymbol,
+            name: "CN",
+            symbol: "CS",
```

Comment:
Can we use more meaningful names here?

Reply:
The same as https://github.com/Cerebellum-Network/chainbridge-ui/pull/131#discussion_r1247759358.